src/components/Header/Header.js

```diff
@@ -1,29 +1,82 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import { Navbar, Nav } from 'react-bootstrap';
+import { NavLink } from 'react-router-dom';
+import { Navbar, Nav, NavDropdown, Container, Row, Col } from 'react-bootstrap';
 
 export default class Header extends Component {
+  isActive({ isCurrent }) {
```

Comment:
This method isn't actually being called anywhere that I can see.

Reply:
Removed